qemu/tests/qemu_guest_agent.py

```diff
@@ -885,6 +885,26 @@ def gagent_check_fsfreeze(self, test, params, env):
         # Finally, do something after thaw.
         self._action_after_fsthaw(test, params, env)
 
+    @error_context.context_aware
+    def gagent_check_thaw_unfrozen(self, test, params, env):
+        """
+        Thaw the unfrozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        try:
+            error_context.context("Verify if FS is thawed", logging.info)
+            expect_status = self.gagent.FSFREEZE_STATUS_THAWED
+            self.gagent.verify_fsfreeze_status(expect_status)
+        except guest_agent.VAgentFreezeStatusError:
+            self.gagent.fsthaw(check_status=False)
+        error_context.context("Thaw the unforzen FS", logging.info)
```

Comment:
I think you may run `gagent.fsthaw()` two times if `verify_fsfreeze_status` failed, do you mean that?

Reply:
Yes, the case is thaw an unfrozen fs, so at first,  I will judge if the fs is thawed, if not, need to thaw it. Then run thaw it again.

Reply:
Thanks, as @vivianQizhu recommended, please add some comment for this case.

Reply:
@sitoliu updated, pls help review again.